user-console/src/main/java/org/pentaho/mantle/client/ui/xul/MantleXul.java

```diff
@@ -154,6 +158,11 @@ public void xulLoaded( GwtXulRunner runner ) {
     burgerToolbarWrapper.setStylePrimaryName( "burgerToolbar-Wrapper" );
     burgerToolbarWrapper.setWidget( burgerToolbar );
 
+    MenuBar tabsMenuBar = ( MenuBar ) container.getDocumentRoot().getElementById( "tabsMenuBar" ).getManagedObject();
+    MenuItem tabsMenuItem = ( MenuItem ) container.getDocumentRoot().getElementById( "tabsMenu" ).getManagedObject();
+    MantleTabPanel.setTabsMenu( tabsMenuBar, tabsMenuItem );
```

Comment:
why is `MantleTabPanel.setTabsMenu( tabsMenuBar, tabsMenuItem );` a static method?

Can't it just be a function on the instantiated object
`tabsMenuBar.setTabsMenu( tabsMenuItem )`

personal preference, I try to avoid static methods at all costs.